src/usecases/createWard.test.js

```diff
@@ -30,4 +36,42 @@ describe("createWard", () => {
       request.hospitalId,
     ]);
   });
+
+  describe("database test", () => {
+    beforeEach(async () => {
+      await truncateAllTables(container);
+    });
+
+    afterEach(async () => {
+      await truncateAllTables(container);
+
+      const db = await container.getDb();
+      db.$pool.end();
+    });
+
+    it("creates a ward in the db when valid", async () => {
+      const trust = await setupTrust(container)({
+        name: "Test Trust",
+        admin_code: "TEST",
+      });
+
+      const hospital = await setupHospital(container)({
+        name: "Test Hospital",
+        trustId: trust.id,
+      });
+
+      const request = {
+        name: "Defoe Ward",
+        hospitalName: "Test Hospital",
+        code: "WardCode",
+        trustId: trust.id,
+        hospitalId: hospital.id,
+      };
+
+      const { wardId, error } = await createWard(container)(request);
+
+      expect(wardId).toBeTruthy;
+      expect(error).toBeNull();
```

Comment:
It might be nice to actually verify that the ward is present in the db rather than trusting that the createWard honestly reports its success.